teamwork/templates/project_details.html

```diff
@@ -2,26 +2,55 @@
 {% load static %}
 
 {% block content %}
+<div id="page-wrapper">
+  <div class="container">
+    <!-- Project details -->
+    <h1>{{ project.name }} <br />
+      <small>{{ project.description }}</small></h1>

```

Comment:
Different html tags should be used for semantics only and css should be used for styling. `<h1>` is meant for headers and everything between that tag is semantically part of the header. Because the description is not part of the header, by best practices, it should be outside of the `<h1>`. So I would make it like this:
`<h1>{{ project.name }}</h1>`
`<div class="project-description">{{ project.description }}</div>`
and then use css to make the description match the preferred style.